extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaJtaConfiguration.java

```diff
@@ -9,7 +9,7 @@
 /**
  *
  */
-@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+@ConfigRoot(name = "narayana", phase = ConfigPhase.RUN_TIME)
```

Comment:
Maybe we should rename the configuration object instead.

Reply:
`transaction-manager` ? or just `transaction`

Reply:
Not sure. We decided to expose Agroal configuration as `datasource` so maybe it makes sense to push this one as `transaction-manager` as Narayana is probably not very well known outside of the WF/EAP world.

@emmanuelbernard @FroMage thoughts?

Reply:
makes sense

Reply:
done